src/frontend/App.scss

```diff
@@ -105,14 +105,19 @@ summary {
   background-color: #fff;
 }
 
-.module-card {
+.module-card, .task-card {
   display: inline-block;
+  width: 100%;
   /* max-width: 18rem; */
   background: #fff;
   margin: 1rem;
   padding: 1rem;
   box-shadow: 0px 1px 4px 0px rgba(0, 0, 0, 30%);
   border-radius: 3px;
+
+  .card-desp{
```

Comment:
Minor: Let's make this `.card-desc` and nest in inside `.module-card, .task-card`